### ISSUE TITLE
Default config options

### ISSUE THREAD

**Issue by skycatminepokie** [Bawnorton/Configurable#3]
Right now (1.1.1), it looks like the YACL defaults are the same as whatever is current (aka the undo button is the same as the reset button).

(this first get call)
```java
public final class YaclScreenFactory {
    public static Screen create(Screen parent, Config config) {
        // ...
       .binding(Binding.generic(config.place_block_clip.default_end_offset.get(), config.place_block_clip.default_end_offset::get, config.place_block_clip.default_end_offset::set))
```

![Example showing the issue](https://github.com/user-attachments/assets/79b4fe43-eec2-4170-aaae-43ef4cfdd091)

**Comment by Bawnorton** [comment]
Is that not the expected behaviour?

**Comment by Bawnorton** [comment]
Ah yeah you're right, I mis-read the docs. I'll change the impl

**Comment by skycatminepokie** [comment]
Awesome, ty! I was struggling to explain it xd

**Comment by Bawnorton** [comment]
Fixed in 1.1.2
